index.js

```diff
@@ -12,8 +12,46 @@ var unlink = RSVP.denodeify(fs.unlink);
 var tmpDir = require('os').tmpdir();
 var debug = require('debug')('async-disk-cache');
 var zlib = require('zlib');
+var heimdall = require('heimdalljs');
 
 var CacheEntry = require('./lib/cache-entry');
+var Metric = require('./lib/metric');
+
+if (!heimdall.hasMonitor('async-disk-cache')) {
+  heimdall.registerMonitor('async-disk-cache', function AsyncDiskCacheSchema() {});
```

Comment:
Empty monitor schemas will be problematic for the heimdall 3 shim. In theory, here we should do something like this.

```js
function AsyncDiskCacheSchema() {
  this.metrics = new Metrics();
}
```

However, if we can use the object that has the counter properties on it directly, we will be in better shape. Is there anything that blocks us from using `Metrics` itself as the Monitor schema?